src/main/java/com/parasoft/demoapp/messages/GlobalPreferencesMessages.java

```diff
@@ -25,6 +25,10 @@ public class GlobalPreferencesMessages extends Messages {
     public static final String LOCALIZATION_LANGUAGE_TYPE_CANNOT_BE_NULL = "Localization language type cannot be null.";
     public static final String ILLEGAL_LABEL_NAME = "Illegal label name: {0}.";
     public static final String LABEL_OVERRIDED_CANNOT_BE_NULL = "LabelOverrided cannot be null.";
+    public static final String ORDER_SERVICE_DESTINATION_QUEUE_CANNOT_BE_NULL = "orderServiceDestinationQueue cannot be null.";
+    public static final String ORDER_SERVICE_REPLY_TO_QUEUE_CANNOT_BE_NULL = "orderServiceReplyToQueue cannot be null.";
+    public static final String INVENTORY_SERVICE_DESTINATION_QUEUE_BE_NULL = "inventoryServiceDestinationQueue cannot be null.";
```

Comment:
Rename INVENTORY_SERVICE_DESTINATION_QUEUE_BE_NULL to INVENTORY_SERVICE_DESTINATION_QUEUE_CANNOT_BE_NULL.

Reply:
All the value of message should be "xxx cannot be null or empty"